aws-lambda-java-events-sdk-transformer/src/main/java/com/amazonaws/services/lambda/runtime/events/transformers/v1/dynamodb/DynamodbStreamRecordTransformer.java

```diff
@@ -13,10 +13,14 @@ public static StreamRecord toStreamRecordV1(final com.amazonaws.services.lambda.
                         DynamodbAttributeValueTransformer.toAttributeValueMapV1(streamRecord.getKeys())
                 )
                 .withNewImage(
-                        DynamodbAttributeValueTransformer.toAttributeValueMapV1(streamRecord.getNewImage())
+                        streamRecord.getNewImage() != null
+                            ? DynamodbAttributeValueTransformer.toAttributeValueMapV1(streamRecord.getNewImage())
+                            : null
                 )
                 .withOldImage(
-                        DynamodbAttributeValueTransformer.toAttributeValueMapV1(streamRecord.getOldImage())
+                        streamRecord.getOldImage() != null
+                            ? DynamodbAttributeValueTransformer.toAttributeValueMapV1(streamRecord.getOldImage())
+                            : null
                 )
```

Comment:
I think this change might be better placed in the method that doesn't handle the `null` input (ie. [`DynamodbAttributeValueTransformer.toAttributeValueMapV1`](https://github.com/aws/aws-lambda-java-libs/blob/311f97cc47b414633c62ee96d9df3e8404e14d69/aws-lambda-java-events-sdk-transformer/src/main/java/com/amazonaws/services/lambda/runtime/events/transformers/v1/dynamodb/DynamodbAttributeValueTransformer.java#L62-L72).

If we add the `null` check there, we can avoid having to remember to handle this everywhere the method is used.

Reply:
Updated. Thanks!